packages/firebase_dynamic_links/README.md

```diff
@@ -106,8 +106,20 @@ Remember not to include `https://` or any slashes or paths in your prefix
   <string>https://example.com/links/share</string>
 </array>
 ```
-
-5. To receive a dynamic link, call the `getInitialLink()` method from `FirebaseDynamicLinks` which gets the link that opened the app (or null if it was not opened via a dynamic link)
+5. If you want to open android app direclty without redirecting to Chrome or Any other Browser you can put an intent-filter in your ```AndroidManifest.xml```. It will open system dialogue to open link with your app or other browsers. Users can directly open link in your app.
+Note:-This step is optional and in case we do not implement this then link will open in chrome at first and then will eventually open your application.
```

Comment:
```suggestion

Note: This step is optional and in case we do not implement this then link will open in chrome at first and then will eventually open your application.
```